backend/petlistings/views.py

```diff
@@ -40,23 +42,26 @@ def petlistings_list_and_create_view(request):
                     return Response({'error': f'Invalid {param}'}, status=status.HTTP_400_BAD_REQUEST)
                 elif param == 'size' and param_value not in ['L', 'M', 'S']:
                     return Response({'error': f'Invalid {param}'}, status=status.HTTP_400_BAD_REQUEST)
-                elif param == 'shelter_email' and not User.objects.filter(email=param_value).exists():
+                elif param == 'shelter_name' and not Shelter.objects.filter(first_name=param_value).exists():
```

Comment:
Can you also add an email option? Right now for shelter profile I'm calling this endpoint with email and want to make sure that it is unique while multiple accounts could have the same name

Reply:
Sure I can add it back in

Reply:
Fixed in [force-push](https://github.com/lujorryn/PetPal-G0346/compare/a927dcb082716a74656fe414036da1ab6b7bdbe1..e2e78cb6eb76fdd3909caf383cec3129300291dd)